src/BgZ-MSZ-1-0/Cert/_reference/resources/zib-TreatmentDirective-bgz-msz-patA-treatmentdir1.xml

```diff
@@ -0,0 +1,113 @@
+<Consent xmlns="http://hl7.org/fhir">
+   <id value="zib-TreatmentDirective-bgz-msz-patA-treatmentdir1"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-TreatmentDirective"/>
+   </meta>
+   <text>
+      <status value="extensions"/>
+      <div xmlns="http://www.w3.org/1999/xhtml">
+         <table>
+            <caption>Toestemming. Patiënt: Dennis D. Dennis Janse. Id: patA-treatmentdir1 (urn:oid:2.16.840.1.113883.2.4.3.11.999.7), Categorie: <span title="behandelaanwijzing (11291000146105 - SNOMED CT)">behandelaanwijzing</span>, Status: actief</caption>
+            <tfoot>
+               <tr>
+                  <td colspan="2">voor behandeling "MRI"; beperking geldt tot aan ontslag uit het ziekenhuis</td>
+               </tr>
+            </tfoot>
+            <tbody>
+               <tr>
+                  <th>Geverifieerd</th>
+                  <td>ja - ${DATE, T, D, +1} - <span title="patiënt (116154003 - SNOMED CT)">patiënt</span>
+                  </td>
+               </tr>
+               <tr>
+                  <th>MRI</th>
+                  <td>
+                     <span title="Ja, maar met beperkingen (JA_MAAR - 2.16.840.1.113883.2.4.3.11.60.40.4)">Ja, maar met beperkingen</span>
+                  </td>
+               </tr>
+               <tr>
+                  <th>Niet toegestaan tenzij</th>
+                  <td>
+                     <div>
+                        <b>Behandelrestrictie</b>
+                        <div>eerst overleg met de partner</div>
+                     </div>
+                  </td>
+               </tr>
+               <tr>
+                  <th>Bron </th>
+                  <td>Levenswensverklaring</td>
+               </tr>
+            </tbody>
+         </table>
+      </div>
+   </text>
+   <extension url="http://nictiz.nl/fhir/StructureDefinition/zib-TreatmentDirective-Verification">
+      <extension url="Verified">
+         <valueBoolean value="true"/>
+      </extension>
+      <extension url="VerifiedWith">
+         <valueCodeableConcept>
+            <coding>
+               <system value="http://snomed.info/sct"/>
+               <code value="116154003"/>
+               <display value="patiënt"/>
+            </coding>
+         </valueCodeableConcept>
+      </extension>
+      <extension url="VerificationDate">
+         <valueDateTime value="${DATE, T, D, +1}"/>
+      </extension>
+   </extension>
+   <extension url="http://nictiz.nl/fhir/StructureDefinition/Comment">
+      <valueString value="voor behandeling &#34;MRI&#34;; beperking geldt tot aan ontslag uit het ziekenhuis"/>
+   </extension>
+   <extension url="http://nictiz.nl/fhir/StructureDefinition/zib-TreatmentDirective-Treatment">
+      <valueCodeableConcept>
+         <coding>
+            <system value="http://hl7.org/fhir/v3/NullFlavor"/>
+            <code value="OTH"/>
+            <display value="Other"/>
+         </coding>
+         <text value="MRI"/>
```

Comment:
In kwalificatiescript staat meer tekst, misschien dat je deze tekst iets wilt uitbreiden

Reply:
Welke tekst bedoel je hier precies? De inhoud van het concept Toelichting is te vinden in de Comment-extension.

Reply:
<html><body>
<!--StartFragment-->

Toelichting (Comment) | voor behandeling "MRI"; beperking geldt tot aan ontslag uit het ziekenhuis
-- | --


<!--EndFragment-->
</body>
</html>

Reply:
Deze is al aanwezig in de FHIR-fixture.